pkg/admission/webhook/generic_webhook.go

```diff
@@ -188,8 +174,10 @@ func (p *WebhookDispatcher) SetHookSource(factory func(cluster logicalcluster.Na
 // SetKcpInformers implements the WantsExternalKcpInformerFactory interface.
 func (p *WebhookDispatcher) SetKcpInformers(local, global kcpinformers.SharedInformerFactory) {
 	p.apiBindingClusterLister = local.Apis().V1alpha1().APIBindings().Lister()
-	p.apiExportIndexer = local.Apis().V1alpha1().APIExports().Informer().GetIndexer()
-	p.globalAPIExportIndexer = global.Apis().V1alpha1().APIExports().Informer().GetIndexer()
+
+	p.getAPIExport = func(clusterName logicalcluster.Name, name string) (*apisv1alpha1.APIExport, error) {
+		return local.Apis().V1alpha1().APIExports().Lister().Cluster(clusterName).Get(name)
```

Comment:
Try local first, then try global

Reply:
Thanks, I saw the indexer did that and wasn't sure if we'd need to with the Lister, too.

Reply:
The lister is backed by the indexer; it provides a more type-safe interface on top